service/ziti-tunnel/service/state.go

```diff
@@ -321,6 +325,10 @@ func (t *RuntimeState) LoadConfig() {
 		log.Warnf("provided mask: [%d] is smaller than the minimum permitted: [%d] and will be changed", rts.state.TunIpv4Mask, constants.Ipv4MinMask)
 		rts.UpdateIpv4Mask(constants.Ipv4MinMask)
 	}
+
+	if t.state.NotificationFrequency == 0 {
```

Comment:
change this to `if t.state.NotificatoinFrequency < rts.UpdateNotificationFrequency(constants.MinimumFrequency)`